src/components/ContentWidgets/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { ContentSingularData } from '../../_data/ContentSingularData';
+import './Gallery.css';
+
+type galleryImagesProps = {
+    images: string[];
+}
+
+const GalleryImages: React.FC<galleryImagesProps> = ({
+    images
+}) => {
+    const COLUMN_NUMBER = 3;
+    var [currentIndex, setCurrentIndex] = useState(0);
+    var [displayState, setDisplayState] = useState(false);
+    const n = Math.ceil(images.length / COLUMN_NUMBER);
+
+    const columnContent = [];
+    for(let col = 0; col < COLUMN_NUMBER; col++) {
+        let rowContent = [];
+        for (let index = n*col; index < n*(col+1); index++) {
+            rowContent.push(<img 
```

Comment:
This should probably be a separate component and use `.map` instead of a for loop but it's probably okay for now.